src/Symfony/Component/Amqp/RetryStrategy/RetryStrategyInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\Amqp\RetryStrategy;
+
+interface RetryStrategyInterface
```

Comment:
not used? 

I mean, ExponentialRetryStrategy and ConstantRetryStrategy implement it, but it seam to be not used elsewhere